pysces/PyscesParse.py

```diff
@@ -237,7 +237,8 @@ class PySCeSParser:
         'abs': 'abs',
         'exp': 'math.exp',
         'ln': 'math.log',
-        'log': 'math.log10',
+        'log': 'math.log',
```

Comment:
We have to be careful here as in MathML 2.0 log without a base is log10 while math.log is ln so I don't think this is a good  ([http://www.zvon.org/comp/r/ref-MathML_2.html#Elements~log](http://www.zvon.org/comp/r/ref-MathML_2.html#Elements~log))